### DISCUSSION TITLE
Block Game Requests For Unsupported Console IDs (and Refund current requests)

### DISCUSSION THREAD

**Discussion by StingX2** [RetroAchievements/RAWeb#820] [Ideas]
This was discussed during the last staff meeting. 

Developers use the most requested games to find projects that people will be happy to receive. It is kinda a huge waste to let users request for systems we don't currently employ. My suggestion was until a system support is revealed (for example psp this past weekend) that requests stay off until that time. Since console ids are assigned already this would be an exclusion of those #s not yet supported.

![image](https://user-images.githubusercontent.com/70493569/132270527-dd8eec9b-a464-4856-bc69-f36924783dd5.png)

The above is a photo showing that Sonic Adventure, Paper Mario, and Sunshine dominate the most requested but the support for these systems is either not ready yet or years away.

![image](https://user-images.githubusercontent.com/70493569/132270460-ba4d3a7c-0c8a-4ec9-9680-25a175f4aaf7.png)

Here is a shot of the Gamecube request list

**Comment by Flashfire42** [comment]
I disagree. It shows what the people really want. 101 people wanting a set for Sonic Adventure means all the more reason to work on dreamcast support. Maybe a better suggestion would be to exclude them from the most requested list with a toggle for systems not yet supported.

**Reply by pinguupinguu** [reply]
I disagree with you partially here. Yes it may show what people want, by why also lead them on with this kind of statistic that can be shown? I feel like having these pages exist to begin with says about our site: "Yeah these pages exist, BUT we don't support the console". To be honest, those pages shouldn't really be there in the first place imo. 

As well, what people want may not really attribute to what's possible developmentally for the cores, so there's also that issue lol. 

I have to go with what Sting is saying here, since console ids can already be flipped on and off if I'm not mistaken.

**Reply by MagmarFire** [reply]
I agree with Pingu and Sting. It's just a waste when people dump requests into games that aren't supported yet, and it clutters it up for people who actually want to glean meaningful information from the page. One might as well ask that games that have sets still show up on the page for historical reasons (not a good idea, in my opinion).

In the end, we need to ask for whom the Most Requested page is really for. The answer is devs, in my opinion, as they're the ones who will actually act on the requests.

**Comment by ScottFromDerby** [comment]
perhaps a checkbox for "Include unsupported systems", disabled by default?

**Reply by Flashfire42** [reply]
I like this idea. Lets people put points where they want and still allows folks to check what games are supported that people want that they can work on

**Comment by televandalist** [comment]
The only reason those pages exist in the first place is because they were reassigned from the many thousands of junk entries that have been made over the years. Had to think of something to do with them. So yeah, that one is my fault :P
